src/ripple/app/ledger/impl/MetaView.cpp

```diff
@@ -218,11 +197,11 @@ MetaView::unchecked_insert(
         using namespace std;
         items_.emplace_hint(iter, piecewise_construct,
             forward_as_tuple(sle->key()),
-                forward_as_tuple(move(sle),
-                    taaCREATE, mSeq));
+                forward_as_tuple(taaCREATE,
+                    move(sle)));
```

Comment:
Dummy check me: the `sle->key()` is guaranteed not to use a moved-from var because `forward_as_tuple` will not invoke a move constructor as the type in the tuple is an rvalue ref. That rvalue ref will then be used in a move constructor one level down, in the constructor of `pair`, but the `sle-key()` call has already happened by then. The code above looks correct, just wanted to reason through it.